src/tools/miri/tests/pass/shims/time-with-isolation2.stdout

```diff
@@ -1 +1 @@
-The loop took around 7s
+The loop took around 12s
```

Comment:
Is this intentional?

Reply:
The test contains a loop over a range of unconstrained integer types which defaults to i32.
Aiui in miri "time" ticks based on the number of mir statements executed or something like that. Since the Range impl changed this also affects rate of time.

Reply:
Argh this test again. We should probably do something a bit better here.

When isolation is enabled (the default), Miri doesn't have access to the system clock, so we use the end of a basic block as an extremely crude approximation of clock ticks, so that we can have a clock when isolation is enabled. This test asserts that the magic number for `NANOSECONDS_PER_BASIC_BLOCK` gets us an isolation clock that's remotely close to the real passage of time. Because when we first did this I think it was off by a factor of 100.

Reply:
I'm still a bit surprised that the number of basic blocks is almost doubling?